src/osvr/Util/LogRegistry.cpp

```diff
@@ -87,22 +87,39 @@ namespace util {
             auto spd_logger = spdlog::get(logger_name);
```

Comment:
Well, and my absolute fallback didn't use any shared pointers, and we should be able to build most of this to use unique_ptr anyway. Unless you are specifying a nothrow copy explicitly, we should be catching here.